Connection/MockedHandler.php

```diff
@@ -27,7 +28,7 @@ private function getResponse(RequestInterface $request): PromiseInterface
         switch ($path) {
             case '/api/citizen':
                 if ('GET' === $method) {
-                    $page = (int) parse_query($request->getUri()->getQuery())['page'];
+                    $page = (int) Query::parse($request->getUri()->getQuery())['page'];
```

Comment:
I'm wondering if this is worth adding new dependency. I've learned over the years that maintaining dependencies is not trivial.

Reply:
the problem is that i installed the release 4.2.1 of mautic, and i had the latest version of guzzlehttp/psr7 and the function `parse_query` was removed

Reply:
I see. I didn't realize it was coming from Guzzle lib. I thought it was from PHP itself. But it is https://www.php.net/manual/en/function.parse-url.php

Then I leave it up to you.